package.json

```diff
@@ -60,7 +60,7 @@
     "webpack-cli": "^6.0.1"
   },
   "dependencies": {
-    "@eppo/js-client-sdk-common": "4.8.1"
+    "@eppo/js-client-sdk-common": "4.8.2"
```

Comment:
don't forget we need to keep this in sync with `node-server-sdk`

Reply:
What do I need to do to make that happen?